docs-src/concepts/what-is-a-list-filter.md

```diff
@@ -11,56 +11,56 @@ tags:
+A List Filter that uses a time range has a resolution of 1 ns on Elasticsearch 7.
+The range of a List Filter timestamp (StartTime, CloseTime, ExecutionTime) cannot exceed 9223372036854775807 (that is, maxInt64 - 1001).
 
-The following is an example List Filter:
+### Supported operators
 
-```
-WorkflowType = "main.YourWorkflowDefinition" and ExecutionStatus != "Running" and (StartTime > "2021-06-07T16:46:34.236-08:00" or CloseTime > "2021-06-07T16:46:34-08:00") order by StartTime desc
-```
+A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute) names, Search Attribute values, and the following supported operators:
 
-[More example List Filters](#example-list-filters)
+- **=, !=, >, >=, <, <=**
+- **AND, OR, ()**
+- **BETWEEN ... AND**
+- **IN**
+- **ORDER BY**
 
-A List Filter contains [Search Attribute](/concepts/what-is-a-search-attribute) names, Search Attribute values, and Operators.
+The `=` operator works like **CONTAINS** to find Workflows with Search Attributes that contain a specific word.
+The **ORDER BY** operator is supported only with [Advanced Visibility](/clusters/how-to-integrate-elasticsearch-into-a-temporal-cluster).
 
-- The following operators are supported in List Filters:
+### Partial string match
 
-  - **=, !=, >, >=, <, <=**
-  - **AND, OR, ()**
-  - **BETWEEN ... AND**
-  - **IN**
-  - **ORDER BY**
-  - **LIKE**
+If a `Description` Search Attribute of `Text` type is set to a phrase, List Filters containing words within that phrase will return the Workflow.
+Custom Search Attributes of `Text` type cannot be used in **ORDER BY** clauses.
 
-- Wildcards ('\*', '%', etc.) are supported for string-type List Filter Search Attributes.
-  Use wildcards with the **LIKE** operator to query possible values:
+- `Description="quick"` or `Description="fox"` would both return a Workflow with the Description Search Attribute set to "The quick brown fox jumps over the lazy dog."
+- `Description="qui"` or `Description="laz"` would not return the Workflow.
+  [Elasticsearch's tokenizer](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-standard-tokenizer.html) is configured to return complete words as tokens.
+- To use a partial string match (i.e. specific word), create a [custom Search Attribute](/app-dev-context/observability#custom-search-attributes) of type `Text` and set it value to a phrase that contains that word.
 
-  ```bash
-  // Create a "ProductId" custom Search Attribute of type String
-  tctl admin cluster add-search-attributes --name ProductId --type String
-  // Match values that start with "book"
-  ProductId LIKE "book%"
-  // Match values that contain "favorite"
-  ProductId LIKE "%favorite%"
-  ```
+### Efficient API usage
 
-- A List Filter applies to a single Namespace.
+An Advanced List Filter API may take longer to respond if it is retrieving a large number of Workflow Executions (over 10 million, for instance).
```

Comment:
I don't know where 10M came from. Server has a max page size setting. By default it is 10000. Most likely original phrase meant total number of workflows in visibility index. But in this case `Count` won't help.

Reply:
Good to know. This was in the original page.